example/movies/ai_assistants.py

```diff
@@ -63,18 +86,23 @@ class MovieRecommendationAIAssistant(AIAssistant):
     id = "movie_recommendation_assistant"  # noqa: A003
     instructions = (
         "You're a helpful movie recommendation assistant. "
-        "Help the user find movies to watch and manage their movie backlogs. "
-        "Use the provided functions to answer questions and run operations.\n"
+        "Use the provided functions to answer queries and run operations.\n"
+        "Use the search function to find movie recommendations based on user's query.\n"
```

Comment:
Had to add this line to force search. In some cases it was hallucinating URLs. For more popular movies, it gets them right, though.

Reply:
> it was hallucinating URLs.

I noticed that too when I tried to use it for the tour guide example.